### PR TITLE
Added 5 new questions to general cs questions

### PR DESCRIPTION
**Pull request by anp-scp**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,49 @@
 const generalCSQuiz = [
+  {
+    Question: "Which of the following is true for adjacency list?",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:3]:
```suggestion
    Question: "Which of the following is true for an adjacency list?",
```

File: src/data/general-cs-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+    Answer: "Edge wieght lookup is O(E).",
+    Distractor1: "Edge weight lookup is O(1).",
+    Distractor2: "Edge weight cannot be specified in adjacency list.",
+    Distractor3: "It cannot be used to represent a cyclic graph.",
+    Explanation: "In adjacency list, for each node only adjacent nodes are stored in a list",
+    Link: "https://www.youtube.com/watch?v=09_LlHjoEiY&list=PLWKjhJtqVAbn5emQ3RRG8gEBqkhf_5vxD&t=682s",
+  },
+  {
+    Question: "Which of the following is true for adjacency matrix?",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:12]:
```suggestion
    Question: "Which of the following is true for an adjacency matrix?",
```

File: src/data/general-cs-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,49 @@
...
+    Answer: "It is space efficient for representing dense graph.",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:13]:
```suggestion
    Answer: "It is space efficient for representing dense graphs.",
```

File: src/data/general-cs-quiz.ts [hunk 4]
```diff
@@ -1,4 +1,49 @@
...
+    Distractor1: "It is space efficient for representing sparse graph.",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:14]:
```suggestion
    Distractor1: "It is space efficient for representing sparse graphs.",
```

**Comment by anp-scp** [comment]
Implemented the requested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**